packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -17,14 +17,19 @@
                 :is-inline="isInline"
                 :data-test-id="testId.label">
                 {{ labelText }}
+                <template #description>
+                    <p
+                        v-if="hasInputDescription"
+                        :class="[
+                            'u-spacingTop',
+                            'u-spacingBottom--large',
+                            $style['c-formField-labelDescription']
```

Comment:
Should be `c-formField-label-description` to match the naming scheme as it's a child element of `c-formField-label`